docs/troubleshooting_deis/index.rst

```diff
@@ -18,16 +18,11 @@ Common issues that users have run into when provisioning Deis are detailed below
 Logging in to the cluster
 -------------------------
 
-Deis runs on CoreOS, so connecting is as simple as using ``ssh``.
-
-CoreOS's default username is ``core``. Use the SSH key you provisioned the cluster with.
-
-Connect to the public IP address of one of your nodes (or use "convenience" DNS records if you've set them up).
+To open a interactive shell on a machine in your cluster:
 
 .. code-block:: console
 
-    $ ssh core@deis-1.example.com -i ~/.ssh/deis
-
+    $ deisctl ssh <machine ID>
```

Comment:
Doesn't `deisctl list` show the shortened machine ID? It seems like we'd need to be able to do this